camposer/src/main/java/com/ujizin/camposer/CameraPreview.kt

```diff
@@ -113,6 +116,7 @@ internal fun CameraPreviewImpl(
     cameraState: CameraState,
     camSelector: CamSelector,
     captureMode: CaptureMode,
+    imageCaptureTargetSize: CameraController.OutputSize?,
```

Comment:
Just a small request, would you mind changing the data type from `CameraController.OutputSize` to `ImageAnalysisTargetSize`? This class is used for `CameraController.OutputSize` mapper for Camposer classes and it would be really helpful!

Also, if you think it would help with clarity, please feel free to change the class name from `ImageAnalysisTargetSize` to `ImageTargetSize`.

Reply:
Yeah, I can do that!